src/data.rs

```diff
@@ -36,6 +36,8 @@ pub struct CPULoad {
     pub system: f32,
     pub interrupt: f32,
     pub idle: f32,
+    #[cfg(target_os = "linux")]
+    pub iowait: f32,
```

Comment:
I didn't say anything about a map. I meant something like this:

```rust
    pub platform: PlatformCPULoad,
}

#[cfg(target_os = "linux")]
pub struct PlatformCPULoad {
    pub iowait: f32,
}

#[cfg(not(target_os = "linux"))]
pub struct PlatformCPULoad {}
```

Accessing `.platform.iowait`  rather than `.iowait` makes it much clearer that you're writing non-portable code and should put `cfg!` around that access.

Reply:
Ah sure - no problem - will do that